haystack/cli/README.md

```diff
@@ -13,32 +13,42 @@ This is the repository where we keep the code for the Haystack CLI.
 
 To contribute to the tutorials please check out our [Contributing Guidelines](./Contributing.md)
 
-## Available commands
+## Available CLI commands
 
 ### `haystack prompt fetch`
 
-```
-Usage: haystack prompt fetch [OPTIONS] [PROMPT_NAME]...
+**Usage:**
 
-  Downloads a prompt from the official Haystack PromptHub and saves it locally
-  to ease use in environments with no network.
+```
+haystack prompt fetch [OPTIONS] [PROMPT_NAME]
+```
 
-  PROMPT_NAME can be specified multiple times.
+Download a prompt from the official [Haystack PromptHub](https://prompthub.deepset.ai/) and save it locally
+for easier use in environments with no network.
 
-  PROMPTHUB_CACHE_PATH environment variable can be set to change the default
-  folder in which the prompts will be saved in.
+- PROMPT_NAME can be specified multiple times.
 
-  If a custom PROMPTHUB_CACHE_PATH is used, remember to also use it for
-  Haystack invocations.
+- PROMPTHUB_CACHE_PATH environment variable can be set to change the default
+folder in which the prompts will be saved in. You can find the default cache path on your machine by running the following code:
 
-  The Haystack PromptHub is https://prompthub.deepset.ai/
+  ``` python
+  from platformdirs import user_data_dir
+  from pathlib import Path
+  Path(user_data_dir("haystack", "deepset")) / "prompthub_cache"
+  ```
 
-Options:
-  --help  Show this message and exit.
-```
+  - If a custom PROMPTHUB_CACHE_PATH is used, remember to also use it for Haystack invocations.
```

Comment:
Great point, @bilgeyucel .  @ZanSara how does one change the path for invocations? Would the user need to change the path here https://github.com/deepset-ai/haystack/blob/a9a49e2c0ae2affe4d604a712737516b4836b449/haystack/nodes/prompt/prompt_template.py#L51 ?

Reply:
No, `PROMPTHUB_CACHE_PATH` is an environment variable, so it's set in the command line. For example on Linux is:

```bash
export PROMPTHUB_CACHE_PATH="/the/new/path/"
```

I think on MacOS is similar and on Windows... i don't really know, it depends on the shell :sweat_smile: I think Windows developers just need to know that they have to set this environment variable in the console and they can usually translate the Linux syntax

Reply:
> If a custom PROMPTHUB_CACHE_PATH is used, remember to also use it for Haystack invocations.

What the line means is something like: "if you set this env var to some specific path in the console while caching the prompts, make sure you also have this variable set to the same value in the console you're using to run your main Haystack instance"

Reply:
Hopefully I understand this right, but how does this look?

> If you set a custom PROMPTHUB_CACHE_PATH environment variable, remember to update it to the same value in the console for your entire Haystack instance.

Reply:
I was sure I commented here yesterday but I don't know where the comment went :D 
So if I'm understanding this correctly, I could add something like this to the readme:

> If you set a custom PROMPTHUB_CACHE_PATH environment variable, remember to update it to the same value in your operating system console.

I'm not sure if we should explain in detail, because "updating env var" is very googleable 🤔 Do you think this would be clear enough for a developer? @bilgeyucel @ZanSara

Reply:
I would go with set instead of update, it's much more common.

Reply:
And if I can nitpick, I'd remove "operating system", it's a bit redundant and might be a little misleading for people using containers

Reply:
Something like: 

> If you set a custom PROMPTHUB_CACHE_PATH environment variable, remember to set it to the same value in your  console before running Haystack.

Reply:
@ZanSara okay, that works well! Just finished updating everything :)